docs/deploying-airbyte/on-kubernetes-via-helm.md

```diff
@@ -0,0 +1,177 @@
+
+## Quickstart
+
+If you don't want to configure your own K8s cluster and Airbyte instance, you can use the free, open-source project [Plural](https://www.plural.sh/) to bring up a K8s cluster and Airbyte for you. Use [this guide](on-plural.md) to get started.
+
+## Getting Started
+
+### Cluster Setup
+
+For local testing we recommend following one of the following setup guides:
+
+* [Docker Desktop \(Mac\)](https://docs.docker.com/desktop/kubernetes)
+* [Minikube](https://minikube.sigs.k8s.io/docs/start)
+  * NOTE: Start Minikube with at least 4gb RAM with `minikube start --memory=4000`
+* [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+For testing on GKE you can [create a cluster with the command line or the Cloud Console UI](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-zonal-cluster).
+
+For testing on EKS you can [install eksctl](https://eksctl.io/introduction/) and run `eksctl create cluster` to create an EKS cluster/VPC/subnets/etc. This process should take 10-15 minutes.
+
+For production, Airbyte should function on most clusters v1.19 and above. We have tested support on GKE and EKS. If you run into a problem starting Airbyte, please reach out on the `#troubleshooting` channel on our [Slack](https://slack.airbyte.io/) or [create an issue on GitHub](https://github.com/airbytehq/airbyte/issues/new?assignees=&labels=type%2Fbug&template=bug-report.md&title=).
+
+### Install `kubectl`
+
+If you do not already have the CLI tool `kubectl` installed, please follow [these instructions to install](https://kubernetes.io/docs/tasks/tools/).
+
+### Configure `kubectl`
+
+Configure `kubectl` to connect to your cluster by using `kubectl use-context my-cluster-name`.
+
+* For GKE
+  * Configure `gcloud` with `gcloud auth login`.
+  * On the Google Cloud Console, the cluster page will have a `Connect` button, which will give a command to run locally that looks like
+
+    `gcloud container clusters get-credentials CLUSTER_NAME --zone ZONE_NAME --project PROJECT_NAME`.
+
+  * Use `kubectl config get-contexts` to show the contexts available.
+  * Run `kubectl config use-context <gke context>` to access the cluster from `kubectl`.
+* For EKS
+  * [Configure your AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) to connect to your project.
+  * Install [eksctl](https://eksctl.io/introduction/)
+  * Run `eksctl utils write-kubeconfig --cluster=<CLUSTER NAME>` to make the context available to `kubectl`
+  * Use `kubectl config get-contexts` to show the contexts available.
+  * Run `kubectl config use-context <eks context>` to access the cluster with `kubectl`.
+
+### Install helm
+
+To install helm simply run:
+
+* For MacOS
+  * `brew install helm`
+* For Linux
+  * Download installer script `curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3`
+  * Assign required premissions `chmod 700 get_helm.sh`
+  * Run script `./get_helm.sh`
```

Comment:
full disclosure I don't linux on the regular, but if we set perms to be `700` won't we need to `sudo ./get_helm.sh` and why not `755`.  that said this is a silly point to make

Reply:
The script will ask you about sudo elevation anyways during the install process
I took the install steps from oficial helm docs
https://helm.sh/docs/intro/install/